packages/moonstone/Scrollable/OverscrollEffect.less

```diff
@@ -0,0 +1,97 @@
+// OverscrollEffect.less
+//
+@import '~@enact/ui/Scrollable/Scrollable.less';
+
+@import '../styles/mixins.less';
+@import '../styles/variables.less';
+@import '../styles/skin.less';
+
+:root {
+	--moon-scrollable-overscroll-ratio-horizontalbefore: 0;
```

Comment:
These CSS variables also don't need to be prefixed with `moon-` since they're part of a moonstone component already. We _sort of_ decided recently that CSS variables defined inside a scope, only need include that scope name for as broad of an audience it's hitting. So, a theme-scoped variable _should_ include both the theme name and the component name it's targeting, but if it's specific to that component, defined only in that component's files, it can just include the component name. Furthermore, if it's defined even narrower than that, like on a child of a component, it can just refer to that name too.

A crummy but ok example:

```
.moonstone {
	--moon-scroller-overscroll-scale: 0.5;

	.scroller {
		--scroller-overscroll-scale: 0.5;

		.overscroll {
			--overscroll-scale: 0.5;
		}
	}
}
```